coverage_point_calculator/src/hexes.rs

```diff
@@ -0,0 +1,90 @@
+use coverage_map::RankedCoverage;
+use hex_assignments::assignment::HexAssignments;
+use rust_decimal::Decimal;
+use rust_decimal_macros::dec;
+
+use crate::{RadioType, Result};
+
+#[derive(Debug, Clone)]
+pub struct CoveredHexes {
+    pub hexes: Vec<CoveredHex>,
+}
+
+#[derive(Debug, Clone)]
+pub struct CoveredHex {
+    pub hex: hextree::Cell,
+    // --
+    pub base_coverage_points: Decimal,
+    pub calculated_coverage_points: Decimal,
+    // oracle boosted
+    pub assignments: HexAssignments,
+    pub assignment_multiplier: Decimal,
+    // --
+    pub rank: usize,
+    pub rank_multiplier: Decimal,
+    // provider boosted
+    pub boosted_multiplier: Option<Decimal>,
+}
+
+impl CoveredHexes {
+    pub fn new_without_boosts(
```

Comment:
I think typically the naming convention here would be `new` and `with_boosts`

Reply:
The logic that used to differentiate these 2 constructors has been moved _into_ `new`. 
Thanks of the help!